hack/install-devtools.sh

```diff
@@ -86,3 +86,10 @@ if [ -z "$(which tparse)" ] || [ "$1" = "--force" ]; then
   echo installing tparse "($VERSION)"
   go install "github.com/mfridman/tparse@$VERSION"
 fi
+
+#--  setup-envtest
```

Comment:
This should be added to the dev-container file as well (`.devcontainer/dev/post-create.sh`)